presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -203,7 +205,7 @@ public void printFinalInfo(Optional<Long> clientStopTimestamp)
                 stats.getProgressPercentage().orElse(0.0));
         out.println(splitsSummary);
 
-        if (debug) {
+        if (runtime) {
```

Comment:
I remember an earlier version which is different--this seems to swap out the runtime for debug flags entirely.  I think you want to leave this, and add a nested check on line 243.

Reply:
Got it.